DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -17,7 +17,12 @@ public interface IEventsSheetsService
     {
         Task AddEventAsync(string name, string description, DateTime time);
         Task EditEventAsync(int key, string? description = null, string? name = null, DateTime? time = null);
-        Task RemoveEventAsync(int key);
+        Task EditEventAsync(
```

Comment:
Did you mean to add a new method? Why not just use the one above? 😕

Reply:
I meant to rename one the arguments (then requiring it to be split across more lines). I think this got snarled up in Git, but I've fixed it now 🙂